pandas/tests/groupby/test_numba.py

```diff
@@ -24,7 +24,8 @@ def test_cython_vs_numba_frame(
             engine="numba", engine_kwargs=engine_kwargs, **kwargs
         )
         expected = getattr(gb, func)(**kwargs)
-        tm.assert_frame_equal(result, expected)
+        check_dtype = func != "sum"
```

Comment:
@jreback looks like `groupby.sum`, unlike `rolling.sum`, coerces the output back to int64 for an int64 input. Currently numba will always return float64. Is this okay for now?

Reply:
let's make an issue to make this consistent in the future.

Reply:
and can you add a reference in the tests for this